physionet-django/environment/deserializers.py

```diff
@@ -13,7 +13,9 @@ def deserialize_research_environments(data: dict) -> Iterable[ResearchEnvironmen
     return [
         ResearchEnvironment(
             id=workbench["id"],
-            dataset=workbench["dataset"],
+            dataset=workbench.get("dataset")
```

Comment:
It's to support previously created workbenches? I wouldn't bother - they won't work anyway (api silently blows up with a 500 after returning "inprogress"). We'll have to only use the new ones.